FWCore/Framework/src/StreamSchedule.cc

```diff
@@ -488,6 +490,26 @@ namespace edm {
         }
       }
     }
+
+    for (auto const& worker : allWorkersLumisAndEvents()) {
+      std::string const& moduleLabel = worker->description()->moduleLabel();
+
+      // The new worker pointers will be null for the TriggerResultsInserter, PathStatusInserter, and EndPathStatusInserter.
+      // We could add special code to create workers for those, but instead we skip them because they do not have
+      // beginStream, endStream, or run/lumi begin/end stream transition functions.
```

Comment:
IIUC this loop adds the workers to `workerManagerBeginEnd_` and `workerManagerRuns_`, based on what is in `workerManagerLumisAndEvents_`. Where exactly does it happen that the worker points for `TriggerResultsInserter` etc are `nullptr`?

Reply:
In getWorker, it first gets the parameter set with ```getPSetForUpdate``` and if that returns nullptr then getWorker returns nullptr. There are not any parameter sets in the configuration for ```TriggerResultsInserter```, ```PathStatusInserter``` and ```EndPathStatusInserter```.

```
    Worker* getWorker(std::string const& moduleLabel,
                      ParameterSet& proc_pset,
                      WorkerManager& workerManager,
                      ProductRegistry& preg,
                      PreallocationConfiguration const* prealloc,
                      std::shared_ptr<ProcessConfiguration const> processConfiguration) {
      bool isTracked;
      ParameterSet* modpset = proc_pset.getPSetForUpdate(moduleLabel, isTracked);
      if (modpset == nullptr) {
        return nullptr;
      }
      assert(isTracked);

      return workerManager.getWorker(*modpset, preg, prealloc, processConfiguration, moduleLabel);
    }
```

Reply:
Thanks for the explanation. Could you extend the first sentence of the comment along "The new worker pointers will be null for the ... **because there are no ParameterSets for those in the configuration**."?

Reply:
Done